### DISCUSSION TITLE
osu!mania R Skin makes lazer default hitsound loop on long notes.

### DISCUSSION THREAD

**Discussion by dtrdpixl** [ppy/osu#30448] [Ideas]
Greetings!

I've been trying a bunch of skins on osu!lazer and realized that on R_skin, the long note hit sound gets replaced by lazer's kick default hit sound on loop, (this never happens in stable) and also makes it sound off sync, which on Full Long Note maps or maps with a bunch of long notes, ends up being incredibly annoying and I would guess it is not the intended function at all.

Video of what happens in-game: https://youtu.be/QFSFVmhsbI8?si=PTvt-LUziV0aelkb

This is the skin that I tried it in first (it happens on most R_skin edits)
https://drive.google.com/file/d/1Xh049bj_aKfQnTojtGupTtWVmfL7A4qC/view

Thanks in advance for seeing this post, and tysm for all your efforts in making the game!

**Comment by peppy** [comment]
Beatmap in video: https://dev.ppy.sh/beatmapsets/953777#mania/1991431

The issue here is that skin's `soft-sliderslider.wav` sound (which has the looped sound you are hearing). It's not actually lazer's default kick. A quick fix would be to delete this file.

As to why and if this should be playing in mania gameplay, I'm not yet sure. Will investigate.

**Reply by peppy** [reply]
Turns out we shouldn't be playing slider slide sounds when a beatmap disables layered sounds (https://github.com/peppy/osu-stable-reference/blob/7519cafd1823f1879c0d9c991ba0e5c7fd3bfa02/osu!/Audio/AudioEngine.cs#L1091).

**Reply by peppy** [reply]
...but neither the beatmap or the skin is disabling layered sounds. This looks like a red herring.

maybe it has something to do with [this weird code](https://github.com/peppy/osu-stable-reference/blob/51bb032bbfebb50055d136a6aed6d83ebb45d9cc/osu!/GameplayElements/HitObjects/Osu/SliderOsu.cs#L1923-L1927)?
